skins/oasis/css/core/ads-porvata.scss

```diff
@@ -2,6 +2,7 @@
 @import 'ads-mixin';
 
 $wikia-top-ads-index: 2;
+$wikia-fullscreen-ads-index: 9999;
```

Comment:
can we add `z-index` instead of `index` ?

Reply:
ok